src/Generating/FinishGen.cpp

```diff
@@ -1783,15 +1783,7 @@ void cFinishGenOreNests::GenerateOre(
 		int BaseY = nestRnd % a_MaxHeight;
 		nestRnd /= a_MaxHeight;
 		// if the NestSize is smaller then four this breaks
```

Comment:
Maybe this comment is not quite up to date anymore.

Reply:
maybe...

Reply:
changed